blog/2021-05-15-announcing-stryker-5-brave-new-dimension.md

```diff
@@ -0,0 +1,141 @@
+---
+slug: announcing-stryker-js-5-brave-new-dimension.md
+title: Announcing StrykerJS 5.0 - Brave new Dimension
+author: Nico Jansen
+author_title: Stryker Team
+author_url: https://github.com/nicojs
+author_image_url: https://avatars3.githubusercontent.com/u/1828233?s=400&u=fec18ad3776aaafec54c49bbd7173a841ae7ea59&v=4
+tags: [stryker]
+---
+
+# Announcing StrykerJS 5.0 - Brave new Dimension
+
+We're proud to announce the next major release of StrykerJS: 5.0. With a name change, a new dimension in your report, a new way to discover files, updated mutators, quality of life improvements, and minor bug fixes.
+
+If you're new to mutation testing, it's a way to measure your tests' effectiveness. A mutation testing framework will make small changes, called _mutants_, one by one in your source code. Then it will run your tests to see if one of them fails. If so, you just "killed" that mutant; if not, it "survived". If too many mutants survive, you probably want to improve your tests. The mutation testing report will give you insides into the test cases you may have missed. If this all sounds complicated, please take a look at [our RoboBar 🤖🍷 example](https://stryker-mutator.io/example).
+
+If you're already using Stryker, you can install this latest version with your package manager of choice.
```

Comment:
Why do I need to install if I'm already using Stryker? Wouldn't I need to update instead?

And shouldn't it be StrykerJS 😬

Reply:
I guess 🙄


😉

Reply:
I'll change it to `npm i -D @stryker-mutator/core@latest`